pkg/driver/clients/http_client.go

```diff
@@ -37,11 +38,9 @@ type MatrixRequest struct {
 	Sets []InputSet `bson:"sets"`
 }
 
-func InvokeDirigent(function *common.Function, runtimeSpec *common.RuntimeSpecification, client *http.Client, isDandelionOptional ...bool) (bool, *mc.ExecutionRecord) {
-	isDandelion := false
-	if len(isDandelionOptional) > 0 {
-		isDandelion = true
-	}
+func InvokeDirigent(function *common.Function, runtimeSpec *common.RuntimeSpecification, client *http.Client, cfg *config.LoaderConfiguration) (bool, *mc.ExecutionRecord) {
+	isDandelion := strings.Contains(cfg.Platform, "Dandelion")
```

Comment:
I insist on that. This doesn't sound the cleanest solution. Why can't you use an = operator here?